iio-private.h

```diff
@@ -249,7 +250,8 @@ struct iio_context * serial_create_context_from_uri(const char *uri);
 
 int local_context_scan(struct iio_scan_result *scan_result);
 
-int usb_context_scan(struct iio_scan_result *scan_result);
+int usb_context_scan(struct iio_scan_result *scan_result,
+		uint16_t scan_usb_vid, uint16_t scan_usb_pid);
```

Comment:
Pass a `const char *arg` to usb_context_scan and extract the VID/PID there. It will be simpler later to merge this change forward to the "dev" branch.

Besides, from a logical standpoint - it's better to have USB-related stuff confined in usb.c.

Reply:
I think I have what you are looking for now.